### DISCUSSION TITLE
Work on follower count functionality

### DISCUSSION THREAD

**Discussion by Njong392** [forem/forem#15683] [Feature Requests]
Hi, DEVelopers. 

My concern is about notification for follower count. A few days ago, I was at six, and all of a sudden, I'm at 23 followers. I wasn't exactly notified after each person followed me. Is it normally supposed to be that way? Because I was thinking that after every follower, then I should get a notification. Just to keep up. Another question, does the email notification alert you only after you've received a batch of followers?

**Comment by barrymichaeldoyle** [comment]
I think they batch the followers increases, I personally wouldn't like a follower notification for every follower when I get 100+ new followers daily, so not too sure this should be a notification per follower for that reason.
